### PR TITLE
classes added by sunnatillo

### PR DESCRIPTION
**Pull request by SunnatSalomov**
please check

### CODE REVIEW

**Review by fayzullayevnodir66 — APPROVED** [review]

File: src/main/java/org/example/jobFestBot/MyBot.java [hunk 1]
```diff
@@ -47,20 +48,20 @@ public void onUpdateReceived(Update update) {
...
-                myExecute(chatId, "choose category", replyMarkup.createInlineKeyboardMarkup((String) null));
+                myExecute(chatId, "choose category", replyMarkup.createInlineKeyboardMarkup(( null)));
             }
         } else if (update.hasCallbackQuery()) {
             String data = update.getCallbackQuery().getData();
             Long chatId = update.getCallbackQuery().getMessage().getChatId();
             Integer messageId = update.getCallbackQuery().getMessage().getMessageId();
-            myEdite(chatId, messageId, " child categories ", replyMarkup.createInlineKeyboardMarkup(data));
-            // myExecute(chatId, createInlineKeyboardMarkup(categories, data));
+            //myEdite(chatId, messageId, " child categories ", replyMarkup.createInlineKeyboardMarkup(data));
```
Comment (axrortatu) [inline src/main/java/org/example/jobFestBot/MyBot.java:63]: why we need comment code if we do not need just remove

**Review by Nodirbek0302 — APPROVED** [review]

**Review by jakhongirzoyirov — APPROVED** [review]

File: src/main/java/org/example/jobFestBot/service/UserService.java [hunk 2]
```diff
@@ -25,8 +26,6 @@ public User add(User user) {
         FileUtils.write(new File(path),valueAsString);
         return user;
```
Comment (axrortatu) [inline src/main/java/org/example/jobFestBot/service/UserService.java:28]: please remove this line as well

File: src/main/java/org/example/jobFestBot/service/UserService.java [hunk 3]
```diff
@@ -60,4 +59,12 @@ public void delete(String id) {
     public User update(User user) {
         return null;
     }
+
+    public boolean existByChatId(Long chatId) {
+        List<User> users = getUser();
+        return users.stream().anyMatch(user -> Objects.equals(user.getChatId(), chatId));
+
```
Comment (axrortatu) [inline src/main/java/org/example/jobFestBot/service/UserService.java:66]: remove empty lines

File: src/main/java/org/example/jobFestBot/service/JobService.java [hunk 4]
```diff
@@ -3,10 +3,9 @@
 import org.example.jobFestBot.model.Job;
 
 import java.util.List;
-import java.util.UUID;
 
 public class JobService implements BaseService<Job> {
-    @Override
+        @Override
```
Comment (axrortatu) [inline src/main/java/org/example/jobFestBot/service/JobService.java:8]: remove unesesarry  space

**Pull request merged**